python/cuspatial/cuspatial/core/binpreds/feature_disjoint.py

```diff
@@ -40,9 +45,8 @@ def _preprocess(self, lhs, rhs):
 
 class PointPolygonDisjoint(BinPred):
     def _preprocess(self, lhs, rhs):
-        intersects = _basic_intersects(lhs, rhs)
         contains = _basic_contains_any(lhs, rhs)
-        return ~intersects & ~contains
+        return ~contains
```

Comment:
These can be one-liners.

```suggestion
        return ~_basic_contains_any(lhs, rhs)
```